doc/en/user/source/extensions/netcdf/netcdf.rst

```diff
@@ -176,6 +176,16 @@ You may specify further custom NetCDF EPSG references by adding more lines to th
 
 #. If the projection wasn't listed, examine the logs for any errors.
 
+Projected Coordinates with axis in km
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For GeoServer < 2.16.x, Projected Coordinates with axis units in km are automatically converted to meters and associated ProjectedCRS has Unit in meters too. Therefore, polygons stored in the geometry table have coordinates in meters.
+
+Starting with GeoServer 2.16.x, automatic conversion km-to-m is disabled by default in order to support km coordinates, directly.
```

Comment:
So if someone were to publish the index as a vector layer, as it's often done for debugging purposes, the CRS would have to be a custom one with units in KM right?

Reply:
yep. In NetCDF CRS are already custom. Till now, all the Projected CRS being built by the NetCDF machinery had units in meters. With this update, by default they will be in KM if not enabling the conversion.

The netcdf.crs. properties of the tests has been indeed updated to also contain the custom KM crs version:
971811=PROJCS["lambert_conformal_conic_1SP", GEOGCS[" ..... UNIT["km", 1000.0], ...  AUTHORITY["EPSG","971811"]]

Reply:
So I'd add at the end ", directly, thus the coordinates in the geometry tables are not in KM and a custom CRS should be used should you want to publish them as a vector layer" or something like that.

Reply:
Documentation updated